includes/class-wp-event-aggregator-eventon.php

```diff
@@ -214,7 +214,7 @@ public function import_event( $centralize_array, $event_args ){
 			 	}
 			}
 
-			if( $centralize_array['organizer']['name'] != '' ){
+			if( isset( $centralize_array['organizer']['name'] ) != '' ){
```

Comment:
It should be if( isset( $centralize_array['organizer']['name'] ) && $centralize_array['organizer']['name'] != '' ){